reliability/single-fit/okhttp/src/test/kotlin/com/datadog/android/okhttp/HeadBasedSamplingTest.kt

```diff
@@ -368,16 +368,16 @@ class HeadBasedSamplingTest {
                     leastSignificantTraceId.toLong().toHexString().padStart(16, '0')
                 )
                 hasMostSignificant64BitsTraceId(mostSignificantTraceId)
-                hasSpanId(spanId.toLong().toHexString())
-                hasParentId(localSpanId)
+                hasSpanId(spanId.cleanDecimalFormat())
+                hasParentId(localSpanId.cleanHexFormat())
                 hasVersion(stubSdkCore.getDatadogContext().version)
                 hasSource(stubSdkCore.getDatadogContext().source)
                 hasTracerVersion(stubSdkCore.getDatadogContext().sdkVersion)
                 hasError(0)
                 hasName("okhttp.request")
                 hasResource("http://${mockServer.hostName}:${mockServer.port}/")
                 hasNoAgentPsr()
-                hasNoSamplingPriority()
+                hasSamplingPriority(2)
```

Comment:
is it a right change? also test name is `parent context = OpenTracing Span, parent sampled`, but there is no `OpenTracing` anymore, so maybe the test is wrong?

Reply:
it's all related to USER_KEEP instead of SAMPLER_KEEP that CoreTracer implementation has instead of that one we had with OpenTracing.